std/fmt.zig

```diff
@@ -958,6 +969,9 @@ test "fmt.format" {
         const value: ?i32 = null;
         try testFmt("optional: null\n", "optional: {}\n", value);
     }
+}
+
+test "fmt.error" {
```

Comment:
The "fmt" on this test case name is now redundant since test names became fully qualified according to file path relative to the package root.